src/Php71/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "symfony/polyfill-php71",
+    "type": "library",
+    "description": "Symfony polyfill backporting some PHP 7.1+ features to lower PHP versions",
+    "keywords": ["polyfill", "shim", "compatibility", "portable"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
+    },
+    "autoload": {
+        "psr-4": { "Symfony\\Polyfill\\Php71\\": "" },
+        "files": [ "bootstrap.php" ]
+    },
+    "minimum-stability": "dev",
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.2-dev"
```

Comment:
That's why I hate aliasing. I didn't modified it as I don't manage releasing cycle of polyfill.
Do you want me to update this in PR? Then it would make sense to update this in other `composer.json` files, would you agree ?